src/@carbon-info/layout/MainLayout/components/Footer/Footer.tsx

```diff
@@ -240,6 +251,14 @@ const useStyles = makeStyles((theme: Theme) => ({
   footNoteText: {
     color: theme.palette.text.footer,
   },
+  altCarbon: {
+    background: "linear-gradient(180deg, #469590 0%, #31D8D6 100%)",
```

Comment:
Add this to `StyleUtils` in `src/@carbon-info/utils/styles.ts` and apply it here.